components/inputs/input-time.js

```diff
@@ -260,5 +264,16 @@ class InputTime extends LitElement {
 			{bubbles: true, composed: false}
 		));
 	}
+
+	async _handleKeydown(e) {
+		console.log(e.keyCode);
```

Comment:
Console :)

Reply:
Question: there are [lint rules](https://eslint.org/docs/rules/no-console) to catch these, is there a reason we don't use them?

Reply:
Not sure... we use `console` a lot in our demos/tests, so that could be it. But yeah, might be worth adding at least in core.

Reply:
I think the eslint recommended rules have changed.  We used to get errors for this, which is why there are opt-out comments in a few places.  

Perhaps we should not rely on the eslint recommended ruleset (we currently extend it, and override where we thought necessary).